tests_integration/nativeapp/test_bundle.py

```diff
@@ -12,21 +12,30 @@
 TEST_ENV = generate_user_env(USER_NAME)
 
 
-# Tests that we disallow polluting the project source through symlinks
-@pytest.mark.integration
-def test_nativeapp_bundle_does_not_create_files_outside_deploy_root(
-    runner,
-    snowflake_session,
-    temporary_working_directory,
-):
+@pytest.fixture
+def template_setup(request, runner, temporary_working_directory):
+    try:
+        template_name = request.param
+    except:
+        template_name = "basic"  # default
     project_name = "myapp"
     result = runner.invoke_json(
-        ["app", "init", project_name],
+        ["app", "init", project_name, "--template", template_name],
         env=TEST_ENV,
     )
     assert result.exit_code == 0
+    return project_name, temporary_working_directory, runner
+
+
```

Comment:
I'd insert a test for the golden path here: just bundle producing what we expect. This is the only test here where I'd expect the template setup to be useful. The other ones don't need the repetition, they check very specific conditions each time.

Reply:
Done. In the new commit, this only checks for the files existing in the deploy root but no their contents. I have added a TODO to replace it with directory snapshot once we have shared utils.